traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/CatalogItemDetails.java

```diff
@@ -171,6 +179,14 @@ public CatalogItemDetails instantiate(JsonReader reader) throws Exception {
                             information = new ArrayMappingFactory<>(new Attribute.AttributeObjectMappingFactory()).instantiate(reader);
                         }
                         break;
+                    case "isWishlisted":
+                        if (reader.peek() == NULL) {
```

Comment:
Dont need to peek here.. if the backend sends `null` here we want to reject the payload and error out, this is a mandatory field, only nullable attributes will require this approach

Reply:
I find that calling /catalog/item/{id} without a travelerId causes isWishlisted to be Null. is this a problem on the backend?
![image](https://user-images.githubusercontent.com/7252786/66220620-53b1ee80-e69b-11e9-9a40-e37c7932614c.png)

Reply:
![image](https://user-images.githubusercontent.com/7252786/66325359-1cd71500-e8f5-11e9-8767-b7199486aa40.png)
as per Bartosz, isWishlited is nullable

Reply:
This conversation isn't resolved unless you have made changes or a discussion has reached somewhere..people involved need know what the resolution was and when you mark as resolved that visibility is no longer

Reply:
But I see your point on this...makes sense